packages/grafana-ui/src/themes/GlobalStyles/legacySelect.ts

```diff
@@ -0,0 +1,205 @@
+import { css } from '@emotion/react';
+
+import { GrafanaTheme2 } from '@grafana/data';
+
+export function getLegacySelectStyles(theme: GrafanaTheme2) {
```

Comment:
Maybe add a comment or note/TODO about having it removed at some point? LegacySelect has been deprecated for years, so we should think about removing it.